src/ci/docker/host-x86_64/dist-various-1/crosstool-ng.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Why was this change made?

Reply:
It was an oversight during rebase. That file is only used for RISC-V builds and it should be moved out of that folder.